cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000074
-name: Median MS1 peak FWHM for peptides
-def: "Median of all MS1 peak widths at half maximum (FWHM) for all identified peptides, in seconds" [PSI:QC]
-comment: Sharper peak widths (i.e. smaller values) indicate better chromatographic resolution.
+name: number of distinct proteins from all PSM
+def: "Number of distinct proteins without duplicate count or groups from all PSM at given FDR threshold." [PSI:QC]
 is_a: MS:4000003 ! single value
-is_a: MS:4000009 ! ID based
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1000086 ! full width at half-maximum
-relationship: has_relation MS:4000013 ! single run based
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000075
-name: Interquartile distance of MS1 peak FWHM for identifications
-def: "Interquartile distance of all MS1 peak widths at half maximum (FWHM) for all identifications, in seconds" [PSI:QC]
-comment: Tighter distributions indicate better consistency.
+name: number of undistinguishable proteins groups from all PSM
+def: "Number of different undistinguishable proteins groups from all PSM at given FDR threshold." [PSI:QC]
 is_a: MS:4000003 ! single value
-is_a: MS:4000009 ! ID based
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1000086 ! full width at half-maximum
-relationship: has_relation MS:4000013 ! single run based
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000076
-name: Median MS1 peak FWHM for peptides in RT quantile
-def: "Median of all MS1 peak widths at half maximum (FWHM) for all identified peptide, in seconds, in the respective identifications' quantiles ordered by retention time. The data is given in a tuple defining the quantiles, 4-tuple represents quartiles, 10-tuple deciles, etc." [PSI:QC]
-comment: Sharper peak widths (i.e. smaller values) indicate better chromatographic resolution in the given quantile.
-is_a: MS:4000004 ! n-tuple
-is_a: MS:4000009 ! ID based
-is_a: MS:4000022 ! chromatogram metric
-relationship: has_relation MS:1000086 ! full width at half-maximum
-relationship: has_relation MS:4000013 ! single run based
+name: ID ratio
+def: "The ratio of identified and recorded MS2 spectra at given FDR threshold." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units STATO:0000300 ! dimensionless ratio
 
 [Term]
 id: MS:4000077
-name: Area under TIC
-def: "The area under the total ion chromatogram." [PSI:QC]
+name: AA sequence length
+def: "The length of an amino acid sequence." [PSI:QC]
```

Comment:
The definition does not add value.
The **number** of amino acids in the sequence?